eth/stubclient.go

```diff
@@ -332,3 +335,84 @@ func (c *StubClient) SetGasInfo(uint64, *big.Int) error { return nil }
 
 // Faucet
 func (c *StubClient) NextValidRequest(common.Address) (*big.Int, error) { return nil, nil }
+
+/* STUB BACKEND
```

Comment:
Let's remove this in the interest of reducing the diff for this PR - if we do end up needing this stub at a later point in time it should be straightforward to add it then.